scarb/src/ops/new.rs

```diff
@@ -133,7 +133,7 @@ fn mk(
 
     // Create the `Scarb.toml` file.
     let manifest_path = canonical_path.join(MANIFEST_FILE_NAME);
-    let edition = edition_variant(Edition::default());
+    let edition = edition_variant(Edition::V2023_10);
```

Comment:
```suggestion
    let edition = edition_variant(Edition:: latest());
```

Replace after https://github.com/starkware-libs/cairo/pull/4380 is merged.

Reply:
Merged

Reply:
I will change it to `latest` after release, as `2.4.0-rc1` is missing this.